engine/consensus/compliance/engine.go

```diff
@@ -90,6 +105,16 @@ func NewEngine(
 	handler := engine.NewMessageHandler(
 		log.With().Str("compliance", "engine").Logger(),
 		engine.NewNotifier(),
+		engine.Pattern{
+			Match: func(msg *engine.Message) bool {
+				_, ok := msg.Payload.(*messages.BlockResponse)
+				if ok {
+					core.metrics.MessageReceived(metrics.EngineCompliance, metrics.MessageBlockResponse)
+				}
+				return ok
+			},
+			Store: pendingBlockResponses,
```

Comment:
Better rename to "range responses", because block responses sounds like responses of each individual block.
```suggestion
			Store: pendingRangeResponses,
```